rpi-eeprom-update

```diff
@@ -597,6 +606,19 @@ lookupVersionInfo()
       BOOTLOADER_UPDATE_IMAGE=""
    fi
 
+   # If the '-s' flag for silent updates is specified then only update the
+   # bootloader if the current version is older than the minimum version.
+   if [ ${SILENT_UPDATE} = 1 ] && [ -n "${BOOTLOADER_AUTO_UPDATE_MIN_VERSION}" ] && [ "${ACTION_UPDATE_BOOTLOADER}" = 1 ]; then
+      if [ "${FIRMWARE_RELEASE_STATUS}" = "critical" ] || [ "${FIRMWARE_RELEASE_STATUS}" = "default" ]; then
+         if [ "${BOOTLOADER_CURRENT_VERSION}" -ge "${BOOTLOADER_AUTO_UPDATE_MIN_VERSION}" ]; then
+            echo "Skip automatic bootloader upgrade. current ${BOOTLOADER_CURRENT_VERSION} >= min ${BOOTLOADER_AUTO_UPDATE_MIN_VERSION}"
+
+            # Clear the update requried flag
+            ACTION_UPDATE_BOOTLOADER=0
```

Comment:
Should this also set `BOOTLOADER_UPDATE_IMAGE=""` like you do inside `checkAndApply` (and on line 606 above) ?